lib/superstore/persistence.rb

```diff
@@ -10,20 +10,24 @@ def find_by_id(id)
       def _insert_record(attributes)
         id = attributes.fetch(primary_key)
 
-        adapter.insert table_name, id, serialize_attributes(attributes)
+        superstore_attributes = serialize_attributes(attributes)
+        adapter.insert table_name, id, superstore_attributes, attributes.except(primary_key, *superstore_attributes.keys)
```

Comment:
Maybe pull arg construction out into an `attributes_for_upsert` method, since it's used twice?

Reply:
Fixed